src/AppBundle/Resources/config/image.yml

```diff
@@ -57,10 +57,15 @@ ezpublish:
                         - { name: strip }
 
 liip_imagine:
+    # global setting, all filters will generate webp images
+    default_filter_set_settings:
+        format: webp
```

Comment:
With this, we're enabling generation of `webp` by default for all filters, even if the source is `jpg` or  `png`? If so, that does not sound right?

Reply:
This is the whole point of the PR. We want to output webp for all web images. Imagick has a possibility of generating webp variations from the source JPG or PNG.
At least that is what Vjeran told me he wants. We don't want to force editors to upload WEBP originals, but we do want to have next-gen webp images served on the frontend. On some other website we had, we used Fastly, Cloudinary or Cloudflare to achieve same effect.

Reply:
There is also an option to configure the liip imagine to generate WEBP only if it is requested in URL (.webp suffix on original image path) and fall back to original type if webp is not requested. This is what Ibexa does by default with their alias paths, however the downside is that you get double the amount of variation images in the storage, which we do not want. Also we do not intend to have fallback formats as all modern browsers have full WEBP support for the past two years.

Reply:
Would we still have a posibility to define different format or from original source formar for some filters?

Reply:
Yes, you can still define desired output format per filter, like in this example
https://symfony.com/bundles/LiipImagineBundle/current/basic-usage.html#client-side-resolving

Also: https://symfony.com/bundles/LiipImagineBundle/current/configuration.html

- default_filter_set_settings - specify the default values that will be inherit for any set defined in filter_sets. These values will be overridden if they are specified in the each set. In case of filters and post_processors, the specified values will be merged with the default ones.
- filter_sets - specify the filter sets that you want to define and use.